### ISSUE TITLE
If access denied to existing group give specific error

### ISSUE THREAD

**Issue by amitaibu** [shoov/shoov-webdrivercss#14]
Currently the error is `Access token is incorrect or no longer valid, visit your account page`

**Comment by pieterdt** [comment]
I get this error while running a test, even though my access token is ok.

**Comment by amitaibu** [comment]
Hi! Yes, right now we have to add you manually to repos you are not the owner. Which repo would you like to be added to ? (You can send me on private if you prefer )

**Comment by pieterdt** [comment]
Hi,

Not entirely clear to me which repo you mean, as I created the test repo
(albeit within our organization account).

Kind regards,

Pieter

**Comment by pieterdt** [comment]
its the repo Wunderkraut-Benelux/swiftcom_testing

**Comment by amitaibu** [comment]
@pieterdt indeed, I see that repo was created by @Tamarpe . Shoov is currently taking a restrictive approach, which will be changed in the future, where it will be better synced with GitHub's access control.

Anyway, I've added you manually via the backend. Let me know if have any other issues :)

**Comment by amitaibu** [comment]
@pieterdt were you able to manage? If you need help, feel free to ping me.

**Comment by pieterdt** [comment]
Hi,

it worked, thx!

Pieter
